microsoft-edge/progressive-web-apps-chromium/how-to/handle-protocols.md

```diff
@@ -11,42 +11,40 @@ keywords: progressive web apps, PWA, Edge, JavaScript, protocol
 ---
 # Handle protocols in Progressive Web Apps
 
-Progressive Web Apps can handle protocols to create a more engaging experience.
+To create a more engaging experience, Progressive Web Apps can handle multiple protocols, such as `http` or `mailto`.
 
-A URL (or Uniform Resource Locator) is a string that allows clients to identify a resource on the Web, and it contains an initial part, the protocol, that lets clients know how to retrieve the resource.
-
-Examples of protocols include:
+A URL (or Uniform Resource Locator) is a string that allows clients to identify a resource on the Web.  The first part of a URL is the _protocol_, which lets clients know how to retrieve the resource.  Examples of protocols include:
 
 | Protocol | Example |
 |:--- |:--- |
 | `http` | http://contoso.com/ |
```

Comment:
@captainbrosset -  the table has `| `http` | http://contoso.com/ |` - I was hoping you'd address that, the source of the revised phrasing of the sentence above it.

Reply:
This table is used to illustrate the general introduction to what URLs and protocols are from the previous sentence. It's not meant to list protocols that PWAs might decide to handle. 
Because it's used as a general introduction, it makes sense for it to list very well known protocols, and http and mailto happen to be (I think) the 2 most well known protocols.

So, while it makes sense to have a small part about what a protocol even is (and use the http example there), it doesn't make as much sense to say that PWAs might want to handle http.

Maybe we should rephrase the start of the article to something like this:

```
A URL (or Uniform Resource Locator) is a string that allows clients to identify a resource on the Web. The first part of a URL is the _protocol_, which lets clients know how to retrieve the resource. Examples of protocols include http or mailto.

To create a more engaging experience, Progressive Web Apps can handle protocols too.
```

So we skip the table. It's a bit too prominent and might give the wrong idea that it's a reference of what protocols PWAs can handle.

And then we go straight to the rest:

```
With Protocol Handling, when a PWA is being installed, the PWA can register to handler certain allowed protocols. ...
```